vintageLines.py

```diff
@@ -27,9 +27,16 @@ def showRelativeNumbers(self):
             name = 'linenum' + str(i-start_line)
             if cur_line == i:
                 num = cur_line+1
+
+                # Line number icons only go up to 999 so if the current
+                # absolute line number exceeds this just use 0.
+                if num > 999:
+                    num = 0
```

Comment:
Maybe num % 1000?

Reply:
I don't quite see how that will be helpful myself - is it not misleading to display e.g. line 1200 as line 200?

Reply:
Whereas displaying 0 for the current line is consistent with vim's default relative number setting.